src/niscope/system_tests/test_system_niscope.py

```diff
@@ -1,527 +1,588 @@
 import collections
 import fasteners
+import grpc
 import hightime
 import math
 import niscope
 import numpy
 import os
+import pathlib
 import pytest
+import subprocess
 import tempfile
-
-
-instruments = ['FakeDevice1', 'FakeDevice2']
-# TODO(sbethur): Use `get_channel_names` when #1402 is fixed
-test_channels = 'FakeDevice2/0,FakeDevice1/1'
-
-
-# There are system tests below that need either a PXI-5124 or a PXI-5142 instead of the PXIe-5164 we use everywhere else
-# because of specific capabilities on those models. Due to internal NI bug 969274, opening a simulated session to those models
-# sometimes fails. As a workaround, the nimi-bot VMs are configured with one persistently simulated instrument of each kind respectively
-# named "5124" and "5142". If you want to run these tests on your own system, you will need to create these two simulated
-# instruments using MAX.
-# In addition, we need a global lock in order to keep us from opening more than one session to the same simulated instrument
-# at the same time. This is because NI-SCOPE (like other MI driver runtimes) disallow two simultaneous sessions to the same
-# instrument, even when the instrument is simulated. This will impact the performance at which system tests run because we
-# parallelize at the tox level :(.
-daqmx_sim_5124_lock_file = os.path.join(tempfile.gettempdir(), 'daqmx_5124.lock')
-daqmx_sim_5124_lock = fasteners.InterProcessLock(daqmx_sim_5124_lock_file)
-daqmx_sim_5142_lock_file = os.path.join(tempfile.gettempdir(), 'daqmx_5142.lock')
-daqmx_sim_5142_lock = fasteners.InterProcessLock(daqmx_sim_5142_lock_file)
-
-
-@pytest.fixture(scope='function')
-def single_instrument_session():
-    with niscope.Session('FakeDevice', False, True, 'Simulate=1, DriverSetup=Model:5164; BoardType:PXIe') as simulated_session:
-        yield simulated_session
-
-
-@pytest.fixture(scope='function')
-def multi_instrument_session():
-    with niscope.Session(','.join(instruments), False, True, 'Simulate=1, DriverSetup=Model:5164; BoardType:PXIe') as simulated_session:
-        yield simulated_session
-
-
-@pytest.fixture(scope='function')
-def session_5124():
-    with daqmx_sim_5124_lock:
-        with niscope.Session('5124') as simulated_session:  # 5124 is needed for video triggering
+import time
+
+
+class SystemTests:
```

Comment:
There's too many changes going on in this file. I can't follow it.
Please utilize 
```
    @pytest.fixture(scope='class')
    def session_creation_kwargs(self, grpc_channel):
        grpc_options = nidcpower.GrpcSessionOptions(grpc_channel, "")
        return {'_grpc_options': grpc_options}
```
like NI-DCPower has done to minimalize changes.

And I think it would also be best if you created the SystemTests classs in a change by itself, like #1846.

Reply:
Good suggestion Jay. Since it looks like there'll be too much for this PR to get in today anyway (also given that Ryan's OOO today), @danestull will look into generating that separate refactoring PR.

Reply:
@DavidCurtiss beat me to it haha See https://github.com/ni/nimi-python/pull/1853

Reply:
Pulled Davids change from main and then added the tests again in a later commit so its easier to see